### PR TITLE
Fix: categories import issue

### PR DESCRIPTION
**Pull request by hamusthekiller**
There is a normal case where wp_set_post_terms doesn't work as expected so, We should return to wp_set_object_terms as per this PR.

### CODE REVIEW

File: includes/class-dropshipping-woocommerce-importer.php [hunk 1]
```diff
@@ -388,10 +388,10 @@ public function get_formatted_product( $product ) {
 							}
 							if ( ! is_wp_error( $new_term ) ) {
 								$term_obj = get_term_by( 'id', $new_term['term_id'] );
-								$cat_id   = wp_set_post_terms( ! empty( $new_product['id'] ) ? $new_product['id'] : 0, $term_obj->slug, 'product_cat', true );
+								$cat_id   = wp_set_object_terms( ! empty( $new_product['id'] ) ? $new_product['id'] : 0, $term_obj->slug, 'product_cat', true );
```
Comment (iamdharmesh) [inline includes/class-dropshipping-woocommerce-importer.php:391]: `wp_set_post_terms` is calling `wp_set_object_terms` internally. it has just few more checks. why we are switching to `wp_set_object_terms`? what issue we are facing with it `wp_set_post_terms`?

**Review by iamdharmesh — CHANGES_REQUESTED** [review]

File: includes/class-dropshipping-woocommerce-importer.php (hunk 1 above)
Comment (suraj-rathod) [reply includes/class-dropshipping-woocommerce-importer.php:391]: Import category is not working here so need to re-verify it

**Comment by suraj-rathod** [comment]
Import category and tags have an issue while import time and on update time it's working

**Comment by iamdharmesh** [comment]
Closed as new PR created for fix https://github.com/Knawat/dropshipping-woocommerce/pull/157

**Pull request closed without merging**